### ISSUE TITLE
Filled Drawers (Void Upgraded) cause issues with Autocrafting

### ISSUE THREAD

**Issue by oguzp1** [raoulvdberge/refinedstorage#1122]
#### Issue description:
Items inside a filled Drawer with a Void Upgrade can be autocrafted to hold the exact amount that the Drawer can handle, but it can be refilled only once before the autocrafting is stuck.
#### What happens:
When you're trying to keep a Drawer with a Void Upgrade filled with the exact amount that the Drawer can handle (the item inside is autocraftable by the RS system), after you pull a couple items from the Drawer it can refill. When you pull more items (again from the Drawer) after that, the system doesn't keep the Drawer filled anymore, and it has a crafting request stuck inside (which is never processed).

Autocrafting mechanism consists of a single Detector which has a Crafter on top. When the Crafter is in "ignore redstone" mode, the crafting is stuck. When it's in "only work with redstone signal" mode, it doesn't get stuck, unless another Crafter in the system has the recipe and is always active.

The system also doesn't seem to be getting stuck if you pull items exclusively from the Grid.
#### What you expected to happen:
Crafters shouldn't get stuck in this case, and shouldn't be forced to work on "only work with redstone signal" mode (even though it's probably more efficient).
#### Steps to reproduce:
1. Basic RS system: Controller, Pattern Grid (with a Pattern), Crafting Monitor, Disk Drive with a Disk with empty space.
2. Drawer: Set to contain Oak Wood Planks, Void Upgraged (Storage Downgrade is optional). Hooked up to an External Storage with higher priority than the Disk Drive.
3. Detector: Set to give a redstone signal whenever Oak Wood Planks are lower than the capacity of the Drawer.
4. Crafter: Contains a single Pattern (Oak Wood -> 4 Oak Wood Planks), on "ignore redstone" mode and "Trigger autocrafting with redstone signal" is set to "Yes". This is placed on top of the Detector.
5. Keep throwing Oak Wood into the system until you fill the Drawer, and then pull some Oak Wood Planks directly from the Drawer. It will not refill.
#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.10.2
- Forge: 12.18.3.2254
- Refined Storage: 1.2.25
- Storage Drawers: 3.6.2

#### Does this issue occur on a server?
Occurs on a singleplayer world.

**Comment by raoulvdberge** [comment]
I don't understand this report. Can you clarify? What do you mean with "When you pull more items (again from the Drawer) after that, the system doesn't keep the Drawer filled anymore, and it has a crafting request stuck inside (which is never processed)."?

**Comment by oguzp1** [comment]
The system stops autocrafting the said item until you cancel that crafting order. For some reason it requests 1 more than it needs to when the Crafter works in "ignore redstone" mode, which it then decides to not craft. And because that crafting request is there, no further requests are taken.

So the basic problem goes like this:
- While the Crafting Monitor shows no active crafting orders, I pull a number of Oak Wood Planks from the drawer.
- The system crafts enough planks to fill the Drawer, voids the excess.
- The Crafting Monitor now shows a crafting request of 1xOak Wood -> 4xOak Wood Planks (the drawer is filled at this point, this is the request that gets stuck).
- I pull out more Oak Wood Planks, and I see no increment in the item count. I can pull out several stacks and it won't craft more.
- I cancel the crafting request, and it starts crafting more. Until it fills up again. There's a new request that gets stuck.
- This repeats.

Configuring the crafter to only work with redstone signal and trigger autocrafting with redstone signal, there seems to be no extra requests getting stuck, the system works as it should (discovered this by accident).

**Comment by raoulvdberge** [comment]
"Trigger autocrafting with redstone signal" is removed and is thus no longer relevant.
